### PR TITLE
feat: inital notes for randomness

### LINKED ISSUE #554: Building Advanced Contracts: randomness section
Create a subsection in "Building Advanced Contracts', providing a contract example requesting a random felt252 using pragma VRF  for some raffle game.

### PR DESCRIPTION
**Pull request by okhaimie-dev**
Closes #554 

## Todo

- [x] Intro topic note
- [ ] Sample code on Raffle Game w. Pragma VRF

- - -
This change is [Reviewable](https://reviewable.io/reviews/cairo-book/cairo-book/662)

### CODE REVIEW

File: src/ch15-05-02-randomness.md [hunk 1]
```diff
@@ -1 +1,59 @@
...
+### Adding Pragma as a Dependency to Your `Scarb.toml` file.
+
+```toml
+[dependencies]
+pragma_lib = { git = "https://github.com/astraly-labs/pragma-lib" }
+```
+
+### Sample Cairo Contract for Randomness: Raffle Game
+
+```rs
```
Comment (enitrat) [inline src/ch15-05-02-randomness.md:19]: can you move this inside a cairo file, as done for all others code snippets in the book, inside the correct listings/ subdirectory?

**Review by enitrat — COMMENTED** [review]
thanks for starting this :)

File: src/ch15-05-02-randomness.md (hunk 1 above)
Comment (okhaimie-dev) [reply src/ch15-05-02-randomness.md:19]: Alright, I'll do in my next commit. 

Waiting on a response from folks from Pragma team to unblock me on their vrf

**Comment by TAdev0** [comment]
@okhaimie-dev  any news from pragma ?

**Comment by okhaimie-dev** [comment]
yes, i will get back to this, later today.

**Pull request closed without merging**